presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -228,6 +234,9 @@
     private final SqlParser sqlParser;
     private final AccessControl accessControl;
 
+    private static Pattern namePattern = Pattern.compile("[a-z_][a-z0-9_]*");
+    private static final String NEW_LINE = System.lineSeparator();
```

Comment:
Remove this and always use "\n" since the SQL output should not depend on the JVM platform line ending